frontend/src/features/admin-form/responses/FeedbackPage/utils/FeedbackCsvGenerator.ts

```diff
@@ -0,0 +1,26 @@
+import moment from 'moment-timezone'
+
+import { FormFeedbackDto } from '~shared/types'
+
+import { CsvGenerator } from '../../common/utils/CsvGenerator'
+
+/**
+ * Class to encapsulate the FeedbackCsv and its attributes
+ */
+export class FeedbackCsvGenerator extends CsvGenerator {
+  constructor(expectedNumberOfRecords: number) {
+    super(expectedNumberOfRecords, 0)
+    this.setHeader(['Date', 'Comment', 'Rating'])
+  }
+
+  /**
+   * Generate a string representing a form feedback CSV line record
+   */
+  addLineFromFeedback(feedback: FormFeedbackDto) {
+    const createdAt = moment(feedback.created)
+      .tz('Asia/Singapore')
+      .format('DD MMM YYYY hh:mm:ss A')
+
+    this.addLine([createdAt, feedback.comment || '', feedback.rating])
```

Comment:
nit: should we use the nullish coalescence operator `??` instead? Using boolean OR may convert a feedback comment of `0` into an empty string.

```tsx
let comment = 0
comment || '' // ''
comment ?? '' // 0
```